pkg/build/strategies/sti/sti.go

```diff
@@ -598,13 +602,11 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 			if config.Quiet {
 				continue
 			}
-			// The log level > 3 forces to use glog instead of printing to stdout
-			if glog.V(3) {
-				glog.Info(text)
-				continue
-			}
-			fmt.Fprintf(os.Stdout, "%s\n", strings.TrimSpace(text))
+			glog.Info(strings.TrimSpace(text))
```

Comment:
Great catch! However, I would be more inclined to achieve this consistency by allowing Info & others to delegate to glog even if V() is not used. My hope being that we can have a near drop in replacement for glog and not require the extra .V() call on all logging statements.